reactioncommerce/reaction
# 2203 Long Strings Handling in Product Option Title

Resolves #2203 

**Testing**
- Clone Branch and run on your Local Machine
- Log In as an admin
- Edit `Variant Option Title` of `Basic Reaction Product` to a longer string like `Red Option is Super Awesome for Everyone`
- Observe that the `title` does not overflow instead it breaks off to the next line

**Screenshot**
<img width="616" alt="screen shot 2017-05-15 at 1 21 50 pm" src="https://cloud.githubusercontent.com/assets/10584060/26057422/85c632b6-3971-11e7-8968-d9c3f752850a.png">
